root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -1930,6 +1930,35 @@ const testData = [
     expected_relationship_type: 'patronage',
             expected_clean_url: 'https://ko-fi.com/35MJZ8OL4IO',
   },
+  // laboiteauxparoles
+  {
+                     input_url: 'https://laboiteauxparoles.com/interprete/269/loco-locass',
+             input_entity_type: 'artist',
+    expected_relationship_type: 'lyrics',
+            expected_clean_url: 'https://laboiteauxparoles.com/interprete/269',
+       only_valid_entity_types: ['artist'],
+  },
+  {
+                     input_url: 'https://laboiteauxparoles.com/titre/55857/wi',
+             input_entity_type: 'work',
+    expected_relationship_type: 'lyrics',
+            expected_clean_url: 'https://laboiteauxparoles.com/titre/55857',
+       only_valid_entity_types: ['work'],
+  },
+  {
+                     input_url: 'http://www.laboiteauxparoles.com/auteur/1682',
+             input_entity_type: 'artist',
+    expected_relationship_type: 'lyrics',
+            expected_clean_url: 'https://laboiteauxparoles.com/auteur/1682',
+       only_valid_entity_types: ['artist'],
+  },
+  {
+                     input_url: 'https://laboiteauxparoles.com/editeur/140',
```

Comment:
Since it drops query part too:

```suggestion
                     input_url: 'https://laboiteauxparoles.com/editeur/140?page=2',
```